services/scraper/search_keyword.go

```diff
@@ -37,38 +26,37 @@ const searchEngineUrl = "https://www.google.com/search?q=%s&lr=lang_en"
 // It will return an error when the collector cannot visit the URL.
 func (service *SearchKeywordService) Run() {
 	collector := colly.NewCollector(colly.Async(true))
-	visitURL := fmt.Sprintf(searchEngineUrl, url.QueryEscape(service.Keyword))
-	searchResult := searchKeywordResult{Keyword: service.Keyword, VisitURL: visitURL}
+	visitUrl := fmt.Sprintf(searchEngineUrl, url.QueryEscape(service.Keyword))
+	keywordResultForm := form.KeywordResultForm{Keyword: service.Keyword, User: service.User}
```

Comment:
My expectation is that forms would be used in the context of controllers solely 🤔 Instead, as per my other comments, calling a service from another service would make more sense in terms of architecture.